plugins/modules/api.py

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+module: api
+
+author:
+  - Tjaž Eržen (@tjazsch)
+
+short_description: Manage ServiceNow POST, PATCH and DELETE requests
+description:
+  - Create, delete or update a ServiceNow record from the given resource.
+  - For more information, refer to the ServiceNow REST Table API documentation at
+    U(https://docs.servicenow.com/bundle/paris-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html#c_TableAPIO).
```

Comment:
```suggestion
    U(https://docs.servicenow.com/bundle/sandiego-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html).
```

Reply:
Release Paris is no longer supported. Link updated to San Diego.

Reply:
Resolved with https://github.com/ansible-collections/servicenow.itsm/pull/179/commits/5972da87f302d46814143365c0651c666cbc265d.